www/includes/easyparliament/templates/html/section/year.php

```diff
@@ -0,0 +1,39 @@
+<div class="full-page__row">
+
+    <div class="business-section">
+        <div class="business-section__header">
+            <h1 class="business-section__header__title">
+                <?= $parent_title ?> &ndash; <?= $title ?>
+            </h1>
+        </div>
+        <div class="business-section__solo">
+            <div class="calendar__controls">
+                <?php if ( isset( $prev ) ) { ?>
+                <a href="<?= $prev['url'] ?>" class="calendar__controls__previous">&larr; <?= $prev['title'] ?></a>
+                <?php } else { ?>
+                <span class="calendar__controls__previous"></span>
+                <?php } ?>
+                <span class="calendar__controls__current"><?= $year ?></span>
+                <?php if ( isset( $next ) ) { ?>
+                <a href="<?= $next['url'] ?>" class="calendar__controls__next"><?= $next['title'] ?> &rarr;</a>
+                <?php } else { ?>
+                <span class="calendar__controls__next"></span>
+                <?php } ?>
+            </div>
+            <?php if ( isset($years) ) {
+                foreach ( $years as $year => $months ) { ?>
+            <div class="calendar-year">
+                <?php foreach ($months as $month => $dates) {
+                  include '_calendar.php';
```

Comment:
The months don't fit correctly always, e.g. ![screen shot 2015-08-11 at 16 32 29](https://cloud.githubusercontent.com/assets/154364/9201798/afae1268-4046-11e5-9559-f4d41d71b3d0.png)